charts/ocis/templates/_common/_tplvalues.tpl

```diff
@@ -97,7 +97,7 @@ oCIS PDB template
 
 */}}
 {{- define "ocis.pdb" -}}
-{{- $_ := set . "podDisruptionBudget" (default (default (dict) .Values.podDisruptionBudget) (index .Values.services .appName).podDisruptionBudget) -}}
+{{- $_ := set . "podDisruptionBudget" (default (default (dict) .Values.podDisruptionBudget) (index .Values.services (split "-" .appName)._0).podDisruptionBudget) -}}
```

Comment:
Where do we have an app name which requires this magic?

Reply:
unfortunately yes.

we append the appNameSuffix here:

https://github.com/owncloud/ocis-charts/blob/c277c52edcd13b6448fbccb6b92cd4065f93cec3/charts/ocis/templates/_common/_tplvalues.tpl#L75

after this, we can't do lookups with the appName to the Values anymore (because this uses the appName without the suffix).

it's ugly, please tell me if you have a better idea.

Reply:
I see